src/main/java/algorithmic_task/service/Impl/SimilarityServiceImpl.java

```diff
@@ -37,8 +38,8 @@ public boolean areSimilar(String sentence1, String sentence2) {
 
         Map<String, Integer> mapDictionary = new HashMap<>();
 
-        for (int i = 0; i < ((List<Set<String>>) dictionary.getDictionary()).size(); i++) {
-            for (String dic : ((List<Set<String>>) dictionary.getDictionary()).get(i)) {
+        for (int i = 0; i < (dictionaryService.getDictionary()).size(); i++) {
+            for (String dic : (dictionaryService.getDictionary()).get(i)) {
```

Comment:
I would like to have method for searching for specific dictionarySet inside dictionaryService not simillarity service.